vendor/github.com/opencontainers/runtime-spec/specs-go/config.go

```diff
@@ -540,14 +550,21 @@ const (
 type LinuxSeccompArg struct {
 	Index    uint                 `json:"index"`
 	Value    uint64               `json:"value"`
-	ValueTwo uint64               `json:"valueTwo"`
+	ValueTwo uint64               `json:"valueTwo,omiempty"`
```

Comment:
You'll probably want to use a runtime-spec commit that includes opencontainers/runtime-spec#884 (which didn't make it into rc6).  We'll have to replace [the version comparison][1] and [generation default][2] with a hard-coded version, probably by defining our own target-spec-version variable in a local version package.  But because runtime-spec versions do not cover their Go library, I think we'd have to do that regardless.

[1]: https://github.com/opencontainers/runtime-tools/blob/0d35fc8437ec8292cd7065e332000c584f7024c5/validate/validate.go#L164-L165
[2]: https://github.com/opencontainers/runtime-tools/blob/0d35fc8437ec8292cd7065e332000c584f7024c5/generate/generate.go#L36